x-pack/plugins/ml/public/application/explorer/explorer_charts/components/explorer_chart_label/entity_filter/entity_filter.tsx

```diff
@@ -59,7 +59,7 @@ export const EntityFilter: FC<EntityFilterProps> = ({
         content={
           <FormattedMessage
             id="xpack.ml.entityFilter.removeFilterTooltip"
-            defaultMessage="Remove filter"
+            defaultMessage="Add negate filter for value"
```

Comment:
We don't seem to have any docs about what a negate filter is, other than perhaps https://www.elastic.co/guide/en/kibana/current/kuery-query.html#_negating_a_query, so my inclination is to try to make it a bit clearer. For example:
```suggestion
            defaultMessage="Add negate filter to exclude the value"
```

Reply:
Discover and Lens use 'Filter out value' here. What do you think about that more concise text @lcawl ?

<img width="358" alt="Screenshot 2024-11-06 at 11 40 46" src="https://github.com/user-attachments/assets/6f721aaa-376d-4056-83f4-9ed8970543c8">

Reply:
updated to use this simplified language in 2400d6127495bc3a5390abb1a804a787a2acea8e
cc @lcawl

Reply:
Thanks, I agree that's clearer.